src/main/java/at/hannibal2/skyhanni/data/IslandType.kt

```diff
@@ -30,11 +36,40 @@ enum class IslandType(val displayName: String) {
     UNKNOWN("???"),
     ;
 
+    var islandData: IslandData? = null
+        private set
+
+    val displayName: String get() = islandData?.name ?: nameFallback
+
+    @SkyHanniModule
     companion object {
+        var islandTypesData: IslandTypeJson? = null
```

Comment:
instead of storing the entire data class as nullable, it should just store the max players and max players mega as non-nullable variables with a fallback if island types data isnt loaded

Reply:
either that or make a fallback island type json